server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -1981,7 +1981,7 @@ private void createDefaultEgressFirewallRule(final List<FirewallRule> rules, fin
 
         // construct rule when egress policy is true. In true case for VR we default allow rule need to be added
         if (!defaultEgressPolicy) {
-            systemRule = String.valueOf(FirewallRule.FirewallRuleType.System);
+            // systemRule = String.valueOf(FirewallRule.FirewallRuleType.System);
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable systemRule was only written to (never read from). Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.